### ISSUE TITLE
Not able to easily copy paste commit hashes in compiler hud

### ISSUE THREAD

**Issue by ezyang** [pytorch/test-infra#4122]
The drop down lists are not copy-pasteable, and the before hash isn't printed anywhere. It would also be nice if the graphs mouseovers would give you hashes as well.

**Comment by huydhn** [comment]
Fixed by https://github.com/pytorch/test-infra/pull/4235.  I have added both the old and the new commits in the summary.  Showing the hash when mouseover the chart is currently blocked by the same payload size limit as https://github.com/pytorch/test-infra/issues/4123.  Basically, the data sent back from Rockset would need to include the hash string for every data points, which would blow up the payload size.  It was one of the optional field that I removed a while ago from the query.  I will create a separate issue to track that feature
